internal/k8sCommon/k8sclient/clientset.go

```diff
@@ -68,7 +68,19 @@ func (c *K8sClient) init() {
 	c.Pod = new(podClient)
 	c.Node = new(nodeClient)
 	// CloudZero replicasets produce high memory consumption when there are large number of replicSets e.g. 71,000
-	c.ReplicaSet = new(replicaSetClient)
+	c.ReplicaSet = nil
+	var TrackRS = os.Getenv("TRACK_REPLICA_SETS")
+	log.Printf("D! CloudZero TraceRS is %v", TrackRS)
+	if TrackRS == "True" || TrackRS == "true" {
```

Comment:
You probably want to use [strconv.ParseBool()](https://pkg.go.dev/strconv#ParseBool)

Reply:
Good idea.   Done!